src/v2/ion/responseTransformer.js

```diff
@@ -229,13 +229,12 @@ const convert = (settings, idx = {}, lastResult = null) => {
   );
 
   injectIdPConfigButtonToRemediation(settings, result);
+  modifyFormNameForIdPAuthenticator(result);
```

Comment:
🤦 🤦 

seems worth to mention orders of function call matters as comments

Reply:
Good idea - will add a comment here!